### PR TITLE
Update app-netutil to account for recent network-resource-injector changes

### PR DESCRIPTION
**Pull request by Billy99**
Network-Resource-Injector added the hugepage pagesize (1G or 2M) and the container
name to the Hugepage Downward API filenames. This was to support containers with mixed
pagesizes and to support multiple containers in a pod, each with hugepage allocation.
Reworked the Hugepage API to account for the changes.

### CODE REVIEW

**Comment by Billy99** [comment]
@zshi-redhat PTAL

File: samples/c_app/README.md [hunk 1]
```diff
@@ -52,14 +52,43 @@ $ LD_LIBRARY_PATH=$PWD/bin:$LD_LIBRARY_PATH PCIDEVICE_INTEL_COM_SRIOV=0000:01:02
...
+include the hugepage size and container name:
+`hugepages_{1G|2M}_{request|limit}_<ContainerName>`
+
+For example:
+```
+sudo sh -c 'echo "1024" >>/etc/podnetinfo/hugepages_1G_request_sriov-example'
+sudo sh -c 'echo "1024" >>/etc/podnetinfo/hugepages_1G_limit_sriov-example'
+```
+
+SR-IOV Network-Resource-Injector also injects an environment variable into
```
Comment (tfherbert) [inline samples/c_app/README.md:70]: the word "also" is confusing and unnecessory

File: samples/c_app/README.md [hunk 2]
```diff
@@ -52,14 +52,43 @@ $ LD_LIBRARY_PATH=$PWD/bin:$LD_LIBRARY_PATH PCIDEVICE_INTEL_COM_SRIOV=0000:01:02
...
+the pod spec with the container's name. This allows the application to process
+hugepage data properly, especially if more than one container in the pod has
```
Comment (tfherbert) [inline samples/c_app/README.md]: comma not necessary

**Review by tfherbert — CHANGES_REQUESTED** [review]
Nice work. A few wording changes in the documentation and comments.

Also, The commit message in the first commit is a bit confusing. Shouldn't it say "Instead of using locally defined const, use the .....

File: samples/c_app/README.md (hunk 2 above)
Comment (Billy99) [reply samples/c_app/README.md]: Removed

File: samples/c_app/README.md (hunk 1 above)
Comment (Billy99) [reply samples/c_app/README.md:70]: Updated the paragraph before to more clearly indicate that NRI is injecting hugepage fields, so now the also indicates NRI also injects an env.

**Comment by Billy99** [comment]
> Also, The commit message in the first commit is a bit confusing. Shouldn't it say "Instead of using locally defined const, use the .....

Reworded commit message.

**Comment by Billy99** [comment]
@zshi-redhat PTAL - Pointed to latest NRI, so ready to merge if you are good with it.

**Review by zshi-redhat — APPROVED** [review]

**Pull request merged**